deps/Makefile

```diff
@@ -183,21 +195,14 @@ compile: $(addprefix compile-, $(DEP_LIBS))
 check: $(addprefix check-, $(DEP_LIBS))
 fastcheck: $(addprefix fastcheck-, $(DEP_LIBS))
 stage: $(addprefix stage-, $(DEP_LIBS_STAGED))
-install: $(addprefix install-, $(DEP_LIBS))
-cleanall: $(addprefix clean-, $(DEP_LIBS))
-distcleanall: $(addprefix distclean-, $(DEP_LIBS))
-	rm -rf $(build_prefix)
-getall: get-llvm get-libuv get-pcre get-openlibm get-dsfmt get-blastrampoline get-openblas get-lapack get-suitesparse get-unwind get-gmp get-mpfr get-patchelf get-utf8proc get-objconv get-mbedtls get-libssh2 get-nghttp2 get-curl get-libgit2 get-libwhich get-zlib get-p7zip get-csl
-
-# If we're building for MacOS, no matter what, `getall` should include `llvmunwind`
-ifeq ($(OS),Darwin)
-getall: get-llvmunwind
-endif
+install: version-check $(addprefix install-, $(DEP_LIBS))
+version-check: $(addprefix version-check-, $(DEP_LIBS_STAGED))
```

Comment:
glad you split this out; this always confused me that it was added in `uninstallers`